### ISSUE TITLE
Unknown optimization bug with multiclass.

### ISSUE THREAD

**Issue by Aylr** [HealthCatalyst/healthcareai-py#427]
dataframe['ThirtyDayReadmitFLG'].replace('Y', 'SnoCones', inplace=True)
dataframe['ThirtyDayReadmitFLG'].replace('N', 'Waffles', inplace=True)
dataframe.loc[0:5, 'ThirtyDayReadmitFLG'] = "Omelette"

classification_trainer = healthcareai.SupervisedModelTrainer(
   dataframe=dataframe,
   predicted_column='ThirtyDayReadmitFLG',
   model_type='classification',
   grain_column='PatientEncounterID',
   impute=True,
   verbose=False)
Yields:

    - LogisticRegression selected performance metrics:
    accuracy: 0.85

		Confusion Matrix (Counts)
		    - Predicted Classes are along the top
		    - True Classes are along the left.

		            SnoCones    Waffles    Omelette
		--------  ----------  ---------  ----------
		SnoCones           0         27           0
		Waffles            0        171           0
		Omelette           0          2           0

**Comment by Aylr** [comment]
Duplicate of #435
